Samples/WinMLSamplesGallery/README.md

```diff
@@ -0,0 +1,18 @@
+﻿# WinML Samples Gallery
+
+![WinML Samples Gallery Screenshot](README_Images/WinMLSamplesGalleryLanding.png)
+
+Demonstrates the different scenarios and features for [WinML](https://docs.microsoft.com/en-us/windows/ai/windows-ml/)
+
+## The WinML Samples Gallery:
+
+- **Demonstrates different scenarios for which WinML excels**: Various use cases that deal with different types of inputs (image, audio, etc.) are demonstrated.
+- **Shows how to implement different scenarios and features**: Each sample comes with a C# file that contains the corresponding code. Code snippets also presented in the app when applicable.
```

Comment:
I don't think it will always be C#. Can we leave that specific part out?

Reply:
Since it's a UWP app, doesn't add a XAML file always come with a corresponding C# file? Unless there's a way to hook up XAML files to other languages that I'm not aware of.

Reply:
I know you can at least add WinUI support to a C++ project, as seen [here](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/simple-winui-example).

Reply:
Ah I wasn't aware. Thanks for letting me know - yes I will leave that part out.